lib/bundle/cask_dumper.rb

```diff
@@ -26,7 +26,7 @@ def dump(casks_required_by_formulae)
     def formula_dependencies(cask_list)
       return [] unless cask_list.present?
 
-      cask_info_response = `brew info --cask #{cask_list.join(" ")} --json=v1`
+      cask_info_response = `brew info --cask #{cask_list.join(" ")} --json=v2`
       cask_info = JSON.parse(cask_info_response)
 
       cask_info.flat_map do |cask|
```

Comment:
```suggestion
      cask_info['casks'].flat_map do |cask|
```

My ruby is rusty so there's probably a better way to do this, but indexing into `casks` seems to resolve the exception for me caused by the different json structure here: https://github.com/Homebrew/homebrew-bundle/issues/819#issuecomment-734007044

Reply:
We could also `.dig("casks")` instead but this is also fine